lib/fserver.js

```diff
@@ -351,7 +351,7 @@
 
 	// start listening for requests
```

Comment:
<!--
  0 failure: 
  1 warning:  lib/fserver.js li...
  
  
  DangerID: danger-id-Danger;
  File: lib/fserver.js;
  Line: 194;
-->

- :warning: lib/fserver.js line 194 – 'url.parse' was deprecated since v11.0.0. Use 'url.URL' constructor instead. (node/no-deprecated-api)